Library/Formula/ace.rb

```diff
@@ -32,5 +32,16 @@ def install
                    "shared_libs=1",
                    "static_libs=0",
                    "install"
+
+    cd "examples" do
+    system "make"
```

Comment:
Sorry, I was possibly unclear there. Just in case, meant:

```ruby
system "make", "-C", "examples"
pkgshare.install "examples"
```

Rather than:
```ruby
cd "examples" do
  system "make", "-C", "examples"
end
pkgshare.install "examples"
```